__tests__/app.test.js

```diff
@@ -245,3 +245,51 @@ describe('GET /api/reviews, optional /:category that filters by category', () =>
     });
   });
 });
+
+describe('GET /api/reviews/:review_id/comments', () => {
+  describe('happy path', () => {
+    test('status 200: should return an array of comments for passed review id with correct keys', () => {
+      const revId = 3;
+      return request(app)
+        .get(`/api/reviews/${revId}/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body.comments;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments.length > 0).toBe(true);
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                review_id: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+  });
+  describe('error handling', () => {
+    test('status 400: should return a message for a bad request', () => {
+      const revId = 'not an id';
+      return request(app)
+        .get(`/api/reviews/${revId}/comments`)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Bad request, not a review id');
+        });
+    });
+    test('status 404: should return a message for an id that does not exist', () => {
+      const revId = 10000;
+      return request(app)
+        .get(`/api/reviews/${revId}/comments`)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe('That review id does not exist');
+        });
+    });
+  });
+});

```

Comment:
I'd recommend 1 more test which gets comments from a valid article_id but there are not any comments for that article, what would you expect this to look like?